tools/scripts/build/build-curriculum.ts

```diff
@@ -14,7 +14,7 @@ const globalConfigPath = path.resolve(__dirname, '../../../shared/config');
 // across all languages.
 void getChallengesForLang('english')
   .then((result: Record<string, unknown>) => {
-    buildExtCurriculumData('v1', result as Curriculum<CurriculumProps>);
+    buildExtCurriculumData('v3', result as Curriculum<CurriculumProps>);
```

Comment:
Sorry if this is explained somewhere, but what happened to `v2`?

Reply:
We can make it v2, we never ended up using v2.